### DISCUSSION TITLE
projectmanager: also be able to change the application's name

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#140] [Ideas]
hi Renaud,

In the project create dialog I thought that the name field was about my own name.  So I gace the app a wrong name.

Now iI can only change the version number and the description. I cannot rename my app. Could that be added?

Folkert

**Comment by renaudpawlak** [comment]
Yes, I will eventually do it. As a workaround, export your project in a file, create a new project with the right name, and import again your file in the new project and save it. Once you are sure it is ok, you can delete the old project with the project manager.
